src/processors/universal.ts

```diff
@@ -370,6 +360,108 @@ class UniversalProcessor extends BaseOrderProcessor {
     return unlockTx;
   }
 
+  private async unlockOrder(
+    orderId: string,
+    order: OrderData,
+    context: OrderProcessorContext
+  ) {
+    if (this.isUnlockBatchLocked) {
+      setTimeout(() => {
+        this.unlockOrder(orderId, order, context);
+      }, 0);
+      return;
+    }
+    const logger = context.logger;
+    this.isUnlockBatchLocked = true;
+    try {
+      logger.info(`Unlocking is started`);
+      const key = `${order.give.chainId}_${order.take.chainId}`;
```

Comment:
Why do we need so complex `key`? Wouldn't it be reasonable to use only `giveChain`? TakeChainId is a constant here.